### PR TITLE
Modal updates cov pools

### PR DESCRIPTION
**Pull request by r-czajkowski**
Ref: #2670 

This PR adjusts all coverage pools modals to the new modal API introduced in #2670.

### CODE REVIEW

File: solidity-v1/dashboard/src/components/modal/coverage-pools/components/WithdrawalOverview.jsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    return "existing withdrawal"
+  }
+
+  const getLabel = () => {
+    if (withdrawalStatus === PENDING_WITHDRAWAL_STATUS.PENDING) {
+      return moment
+        .unix(withdrawalInitiatedTimestamp)
+        .add(withdrawalDelay, "seconds")
+        .format("MM/DD")
+    } else if (withdrawalDelay === PENDING_WITHDRAWAL_STATUS.COMPLETED) {
```
Comment (michalsmiarowski) [inline solidity-v1/dashboard/src/components/modal/coverage-pools/components/WithdrawalOverview.jsx:161]: Should be `withdrawalStatus` here

```suggestion
    } else if (withdrawalStatus === PENDING_WITHDRAWAL_STATUS.COMPLETED) {
```

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left some comments, but overall good job!

File: solidity-v1/dashboard/src/components/modal/coverage-pools/components/WithdrawalOverview.jsx (hunk 1 above)
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/coverage-pools/components/WithdrawalOverview.jsx]: 68f104f

File: solidity-v1/dashboard/src/components/modal/coverage-pools/deposit/InitiateDeposit.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                <span className="mr-a">CovKEEP Received</span>
+                {covKEEP.displayAmountWithSymbol(covKEEPReceived)}
+              </List.Item>
+            </OnlyIf>
+          </List.Content>
+        </List>
+      </ModalBody>
+      <ModalFooter>
+        <OnlyIf condition={!transactionHash}>
+          <form>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/coverage-pools/deposit/InitiateDeposit.jsx]: 88d5760

File: solidity-v1/dashboard/src/components/modal/coverage-pools/IncreaseWithdrawal.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          <Banner.Description
+            className="text-black text-bold"
+            style={{ margin: "0", marginLeft: "0.5rem", fontSize: "0.875rem" }}
+          >
+            The cooldown period is 21 days.
+          </Banner.Description>
+        </Banner>
+      </ModalBody>
+      <ModalFooter>
+        <form>
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/coverage-pools/IncreaseWithdrawal.jsx]: 88d5760

File: solidity-v1/dashboard/src/components/modal/coverage-pools/deposit/InitiateDeposit.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+          </form>
+          <SubmitButton
+            className="btn btn-primary btn-lg mr-2"
+            type="submit"
+            onSubmitAction={(awaitingPromise) => {
+              dispatch(depositAssetPool(amount, awaitingPromise))
+            }}
+            disabled={!(formik.isValid && formik.dirty)}
+          >
+            Deposit
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/coverage-pools/deposit/InitiateDeposit.jsx]: 5c01248

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Pull request merged**